packages/babel-plugin/__tests__/legacy/transform-logical-values-test.js

```diff
@@ -128,74 +128,6 @@ describe('@stylexjs/babel-plugin', () => {
       `);
     });
 
-    /**
-     * Non-standard values
```

Comment:
These got moved to the 'polyfills' test file. But the goal is to be able to delete these once there is appropriate compiler validation.